lib/launcher.rb

```diff
@@ -4,6 +4,8 @@
 require 'proxy/signal_handler'
 require 'proxy/log_buffer/trace_decorator'
 require 'thread'
+require 'rack'
+require 'webrick'
```

Comment:
Webrick is still required dependency, it will never be optional. We will just remove it after couple of releases for once and forever from the codebase (all the hacks, monkey patches, the configuration value and everything else including this require).